x-pack/plugins/alerting/server/lib/snooze/get_active_snooze_if_exist.ts

```diff
@@ -10,7 +10,7 @@ import { RuleSnoozeSchedule } from '../../types';
 
 const MAX_TIMESTAMP = 8640000000000000;
 
-export function isSnoozeActive(snooze: RuleSnoozeSchedule) {
+export function getActiveSnoozeIfExist(snooze: RuleSnoozeSchedule) {
```

Comment:
The function returns an active snooze or `null`. I found the `isSnoozeActive` misleading and I renamed the naming. I am open to suggestions 🙂.